src/vos/README.md

```diff
@@ -105,79 +102,66 @@ For example, when an array object is "punched" from 0 to infinity in a given epo
 
 Internally, the VOS maintains an index of container UUIDs that references each container stored in a particular pool.
 The container itself contains three indices.
-The first is an index of objects used to map object IDs to object metadata efficiently when servicing I/O requests.
-The second index enumerates all object updates by epoch for efficient discard on abort and epoch aggregation.
-The third index maps container handle cookies, which identify updates from a process group to object IDs within that container.
-The container handle cookie is used for identifying object IOs associated with a process group and primarily would be used for aborting a transaction associated with a process group from DAOS-M.
+The first is an object index used to map an object ID and epoch to object metadata efficiently when servicing I/O requests.
+The other two indicies are for maintining active and committed <a href="#811">DTX</a> records for ensuring efficient updates across multiple replicas.
+
+DAOS supports two types of values, each associated with a Distribution Key (DKEY) and an Attribute Key (AKEY): Single value and Array value.
+An update to a single value replaces the entire value whereas an array update only replaces the records specified in the update.
```

Comment:
Maybe a comment about how an entire single value must be fetched ... can't fetch just parts of a single value.